src/live_restore/live_restore_server.c

```diff
@@ -362,17 +379,19 @@ __wt_live_restore_server_create(WT_SESSION_IMPL *session, const char *cfg[])
         return (0);
 
     WT_CONNECTION_IMPL *conn = S2C(session);
-    WT_ERR(__wt_calloc_one(session, &conn->live_restore_server));
 
     /* Read the threads_max config, zero threads is valid in which case we don't do anything. */
     WT_CONFIG_ITEM cval;
     WT_ERR(__wt_config_gets(session, cfg, "live_restore.threads_max", &cval));
     if (cval.val == 0)
         return (0);
 
+    WT_ERR(__wt_calloc_one(session, &conn->live_restore_server));
     WT_ERR(__wt_spin_init(
       session, &conn->live_restore_server->queue_lock, "live restore migration work queue"));
 
```

Comment:
Add a comment here that we're only setting this for clarities sake.

Reply:
[95e1356](https://github.com/wiredtiger/wiredtiger/pull/11797/commits/95e13564e6f9381ee83800b64bcf691208ead716)